### ISSUE TITLE
Focus indication is inconsistent cross-browser

### ISSUE THREAD

**Issue by axamei** [pulibrary/orangelight#1224]
Firefox:
<img width="61" alt="screen shot 2018-03-26 at 9 56 15 am" src="https://user-images.githubusercontent.com/8161144/37910616-f694515a-30db-11e8-8cad-76e6a2429b6f.png">
Chrome:
<img width="78" alt="screen shot 2018-03-26 at 9 55 57 am" src="https://user-images.githubusercontent.com/8161144/37910620-f7f380f2-30db-11e8-8777-f346bbedf9ba.png">

Focus indicator should be consistent. Ensure blue border/highlight is present cross-browser to indicate focus.
